README.md

```diff
@@ -9,10 +9,17 @@ Version 2.0 of Leaflet.awesome-markers is tested with:
 
 For bootstrap 2.x & Fontawesome 3.x use Leaflet.awesome-markers v1.0
 
+Version 1.0 of Leaflet.awesome-markers is tested with:
+- Bootstrap 2
+- Font Awesome 3.0
+- Leaflet 0.5
+
+Use version 2.0 for newer versions of bootstrap and font-awesome!
```

Comment:
Use version 2.0.x for compatibility with newer versions of bootstrap, font-awesome and leaflet.
It is tested and works with the latest available versions:
- Bootstrap 3.3.7
- Font Awesome 4.7.0
- Leaflet 1.0.3